contracts/Orchestrator.sol

```diff
@@ -0,0 +1,165 @@
+pragma solidity 0.4.24;
+
+import "openzeppelin-eth/contracts/ownership/Ownable.sol";
```

Comment:
Since this contact is not upgradable, I'd much rather use the non-upgradable openzeppelin contract libraries.
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol

Reply:
I tried that initially, but switched back to the ZeppelinOS version. The issue was that two contracts were being defined with the same name.

An alternative approach would be to copy Ownable locally, but call it something else and use that one.